VeganVirus/VeganVirus/DesktopAction.cpp

```diff
@@ -0,0 +1,93 @@
+#include "DesktopAction.h"
+#include <cmath>
+
+#define BOUND_DIFF 10
+
+DesktopAction::DesktopAction(Draw* draw, MouseManager& mouseManager, DesktopManager& desktopManager)
+	:Action(DESKTOP_ACTION_PROGRESS, L"meat.ico"), _mouseManager(mouseManager), _desktopManager(desktopManager)
```

Comment:
Put the action's icon in a `#define`. Also, add a new image to the project (in the same file as the code), and add it as a resource in visual studio.

```suggestion
	:Action(DESKTOP_ACTION_PROGRESS, DESKTOP_ACTION_ICON), _mouseManager(mouseManager), _desktopManager(desktopManager)
```